examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py

```diff
@@ -190,8 +189,9 @@ def __transform_DFTB_UV_input_to_data_object_base(self, raw_data_path, dir):
         data_object = None
 
         # collect information about molecular structure and chemical composition
+        #print ("__transform_DFTB_UV_input_to_data_object_base:", raw_data_path, dir)
         try:
-            pdb_filename = raw_data_path + '/' + dir + '/' + 'smiles.pdb'
+            pdb_filename = dir + '/' + 'smiles.pdb'
```

Comment:
Should we just remove `raw_data_path` from the function signature?

Reply:
You are right.  `raw_data_path` is not needed anymore. I will remove.